test/parallel/test-debugger-pid.js

```diff
@@ -50,5 +50,5 @@ interfacer.on('line', function(line) {
 });
 
 interfacer.on('exit', function(code, signal) {
-  assert.ok(code == 1, 'Got unexpected code: ' + code);
+  assert.strictEqual(code,1, `Got unexpected code: ${code}`);
```

Comment:
Another tiny nit: Can you leave a space before the `1`? (I’m not sure but running `make lint` might help catch that…?)